concordium-sdk/src/main/java/com/concordium/sdk/responses/transactionstatus/CredentialKeysUpdatedResult.java

```diff
@@ -1,19 +1,23 @@
 package com.concordium.sdk.responses.transactionstatus;
 
+import com.concordium.sdk.transactions.CredentialRegistrationId;
 import com.fasterxml.jackson.annotation.JsonCreator;
 import com.fasterxml.jackson.annotation.JsonProperty;
 import lombok.Getter;
 import lombok.ToString;
 
+import java.util.Objects;
+
 @Getter
 @ToString
 public final class CredentialKeysUpdatedResult extends TransactionResultEvent {
-    private final String credId;
+    private CredentialRegistrationId credId;
 
     @JsonCreator
     CredentialKeysUpdatedResult(@JsonProperty("credId") String credId) {
-        this.credId = credId;
-
+        if(!Objects.isNull(credId)) {
+            this.credId = CredentialRegistrationId.from(credId);
+        }
```

Comment:
I don't understand this check. In what case will the string be null? And why is it OK to do nothing then, as opposed to raising an exception?